### PR TITLE
Add more info about the last stretch.

### PR DESCRIPTION
**Pull request by micha7a**
I tried to add information from the email from Doctoral students’ services. 
Please check if what I added is understandable.

### CODE REVIEW

File: last/_posts/2021-08-30-timeline.md [hunk 1]
```diff
@@ -11,7 +11,11 @@ order: 3
...
-1. Your public defence can be any time (up to 6 months) after your private one (given you have passed the private one)
-1. For non-EU people it may make sense to look for a job before your public defense, to avoid worries about your permit expiring
-1. It seems that there is not so much work between the defences, but you might have corrections from the jury to apply and the presentation to prepare, so having it the next day might not be ideal ;) 
+2. After your private defence, remember to send the form to SAC (Doctoral students’ services, not EDIC). You should get from them an email with further details. 
+3. Your public defence can be any between one and six months after your private one (given you have passed the private defence)
+4. After the private defence, you need to upload the final version of your thesis (and get your supervisors approval) 15 days before the public defence. This is necessary because to get the diploma you need to first aprove the proof (a test print) of your thesis. Repro needs a couple of days to prepare the proof, but the 15 days is to be on the safe side. 
+5. You also need to infrom SAC about your public defence at leat 15 days before it.
```
Comment (SolalPirelli) [inline last/_posts/2021-08-30-timeline.md:17]: Typo: infrom -> inform

File: last/_posts/2021-08-30-timeline.md [hunk 2]
```diff
@@ -11,7 +11,11 @@ order: 3
...
+3. Your public defence can be any between one and six months after your private one (given you have passed the private defence)
+4. After the private defence, you need to upload the final version of your thesis (and get your supervisors approval) 15 days before the public defence. This is necessary because to get the diploma you need to first aprove the proof (a test print) of your thesis. Repro needs a couple of days to prepare the proof, but the 15 days is to be on the safe side. 
+5. You also need to infrom SAC about your public defence at leat 15 days before it. 
+6. For non-EU people it may make sense to look for a job before your public defense, to avoid worries about your permit expiring
+7. It is up to your supervisor how long you contract will last. In most cases, your contract can be extended moth or more after you finish your PhD. You can then help younger PhDs and search for a job comfortabely. However, EPFL only require that you are paid until your **private** defence, so you might want to discuss that with your supervisor.
```
Comment (SolalPirelli) [inline last/_posts/2021-08-30-timeline.md:19]: Typo "moth" and missing a number I guess?
Also I think you mean as scientific assistant or postdoc? Would be good to clarify, esp. for students whose advisor is new and may need to be told about this

**Comment by SolalPirelli** [comment]
It's understandable to me, just some minor typos ^^

File: last/_posts/2021-08-30-timeline.md (hunk 2 above)
Comment (micha7a) [reply last/_posts/2021-08-30-timeline.md:19]: TBH, I have no idea, on the website it was still "Doctoral Assistant" for me :/ I've heard that you can't get a postdoc salary right after your defence. Not sure whom we could ask about that.

**Pull request merged**